File: python/python_secao8/Exercicio03.py
#variáveis
vetor = []

#entradas
for i in range(0, 10):
    num = int (input("Informe o valor do número:"))
    vetor.append(num)
# processamento
vetor.reverse() # inverte a lista 
for i in vetor:
    print(i)

File: java/secao6/Exercicio03.java
package br.com.geekuniversity.secao6;

import java.util.Scanner;

public class Exercicio03 {

	public static void main(String[] args) {
		//Variáveis
		int numero, p=0, i=0;
		Scanner teclado = new Scanner(System.in);
		//entradas 
		System.out.println("Informe o número:");
		numero = teclado.nextInt();
		//processamento
		if (numero % 2 == 0) {
			p = numero;
		} else {
			i = numero; 	
		}
		//saída
		System.out.println(p + " é par!!");
		System.out.println(i + " é ímpar!");
		
		teclado.close();
		}
	    
	    }
